src/components/Bookmark.spec.js

```diff
@@ -0,0 +1,51 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import Bookmark from './Bookmark'
+
+describe('Bookmark', () => {
+  it('has a bookmark image', () => {
+    render(<Bookmark />)
+
+    const img = screen.getByRole('img')
+
+    expect(img).toBeInTheDocument()
+  })
+
+  it('has an alt attribute', () => {
+    render(<Bookmark />)
+
+    const alt = screen.getByAltText('bookmark')
+
+    expect(alt).toBeInTheDocument()
+  })
+
+  it('has an inactive button', () => {
```

Comment:
This is not a good description of the test. When reading this, I would expect a test which covers whether the `button` is disabled or not.